markdown/3.x/en/ff-asn.md

```diff
@@ -357,3 +357,17 @@ This template has to contain exactly one `input` element.
         </div>
     </ff-asn-group>
 ```
+
+### Dispatching data manually
```

Comment:
I don't see manual data dispatching as part of _Searchable ASN groups_ so I think it's best to increase the headline level.
```suggestion
## Dispatching data manually
```

There is also an indentation error on **line 322** messing up the displayed formatting in GitHub. Would you mind fixing that with the other changes?

Reply:
Indentation has been removed